assets/js/modules/analytics/datastore/index.js

```diff
@@ -45,9 +50,15 @@ const baseModuleStore = Modules.createModuleStore( 'analytics', {
 	adminPage: 'googlesitekit-module-analytics',
 } );
 
+const existingTagStore = createExistingTagStore( 'modules', 'analytics', {
+	tagMatchers,
+	isValidTag: isValidPropertyID,
+} );
+
```

Comment:
See above, let's move this to the `tags.js` file where the similar functionality was before. It's more of an implementation detail that it uses `createExistingTagStore`, so I think we should keep that where it originally was.